add case studies instruction

dudashu opened the pull request:
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)


* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

docs update

* **What is the current behavior?** (You can also link to an open issue here)

add instruction for case studies with PR #441

fisherxu commented on docs/community/casestudies.md:
```diff
@@ -0,0 +1,43 @@
+
+After completing your case study write-up, please submit your case document to the KubeEdge [website](https://github.com/kubeedge/website) repository, and it will be displayed in the official website's case studies page after community review.
+
+1.  Create a directory for your case study under `src/pages/case-studies` and `i18n/zh/docusaurus-plugin-content-page/case-studies`, naming the directory after your case.
```
```suggestion
1.  Create a directory for your case study under `src/pages/case-studies` in English or `i18n/zh/docusaurus-plugin-content-page/case-studies` in Chinese, naming the directory after your case.
```

fisherxu commented on docs/community/casestudies.md:
```diff
@@ -0,0 +1,43 @@
+After completing your case study write-up, please submit your case document to the KubeEdge [website](https://github.com/kubeedge/website) repository, and it will be displayed in the official website's case studies page after community review.
+
+1.  Create a directory for your case study under `src/pages/case-studies` and `i18n/zh/docusaurus-plugin-content-page/case-studies`, naming the directory after your case.
+2.  Place your English case document under the directory you created in `src/pages/case-studies/`, and place your Chinese case document under the corresponding directory you created in `i18n/zh/docusaurus-plugin-content-page/case-studies`.
```
```suggestion
2.  Place your English case document under the directory you created in `src/pages/case-studies/`, or place your Chinese case document under the corresponding directory you created in `i18n/zh/docusaurus-plugin-content-page/case-studies`.
```

fisherxu commented on docs/community/casestudies.md:
```diff
@@ -0,0 +1,43 @@
+
+# Submit Your Case
+
+After completing your case study write-up, please submit your case document to the KubeEdge [website](https://github.com/kubeedge/website) repository, and it will be displayed in the official website's case studies page after community review.
```
```suggestion
After completing your case study write-up, please submit your case document to the KubeEdge [website](https://github.com/kubeedge/website) repository, you can submit your case in English or Chinese(languages ​​currently supported by the website), and it will be displayed in the official website's case studies page after community review. 
```

fisherxu reviewed (commented):
We should welcome users to submit English or Chinese cases they want :)

dudashu commented:
> We should welcome users to submit English or Chinese cases they want :)

Thanks for your suggestions. I have modified it and also modified the Chinese docs.

fisherxu reviewed (commented):
/approve

Looks good.

Shelley-BaoYue commented:
/lgtm
